service/src/main/resources/db/changesets/20240412-testdata.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
```

Comment:
hmm i wonder if there something we can do with test data stuff so we can keep all of the test seeding changes in one migration file 🤔 .  Im curious what would happen if you put these changes in the other test data file and ran against an already existing database.

Reply:
agreed it'd be nice to keep them in one file. I'll try.

Reply:
I realized we can always keep just one testdata file, since it's only used in the test context, so live environments and bees and our local env won't ever apply them.

Reply:
i did notice that my local environment did have a reference to the original test migration file (which may be wrong because i have accidentally ran tests against my local environment lol). Just wanted to double check that your local environment didnt get upset when changing the name of the original test data file.  It should never apply any of the changes in the file but im not sure if it keeps a reference to it / needs to have that reference exist from then on.

Reply:
I tried in my BEE and it worked fine so I think it's good. my local environment got messed up when I changed the name of the table from pipeline_inputs_definitions to pipeline_input_definitions so I had to blow that away but otherwise it worked fine